MANIFEST.in

```diff
@@ -7,3 +7,5 @@ exclude pyproject.toml
 # Note that README.rst is included in a source distribution by default:
 # https://packaging.python.org/guides/using-manifest-in/#how-files-are-included-in-an-sdist
 include README_SKINNY.rst
+include requirements/skinny-requirements.txt
+include requirements/core-requirements.txt
```

Comment:
Do we need to include these files in the package?

Reply:
yeah, else `setup.py` can't read them during wheel installation

Reply:
Makes sense :)